content/en/docs/distributions/openshift/install-kubeflow.md

```diff
@@ -40,8 +40,7 @@ Minimal:
 Use the following steps to install Kubeflow 1.3 on OpenShift 4.x.
 
 
-1. Download the example "kfdef" for Kubeflow 1.3 on Openshift from [kubeflow/manifests/distributions/kfdef]
-(https://raw.githubusercontent.com/opendatahub-io/manifests/v1.3-branch/distributions/kfdef/kfctl_openshift_v1.3.0.yaml).
+1. Download the example "kfdef" for Kubeflow 1.3 on Openshift from [kubeflow/manifests/distributions/kfdef](https://raw.githubusercontent.com/opendatahub-io/manifests/v1.3-branch/distributions/kfdef/kfctl_openshift_v1.3.0.yaml).
```

Comment:
I think its a bit confusing to link directly to the raw YAML (it removes the context of the repo).

I think its easier to understand if the link is:
```
https://github.com/kubeflow/manifests/blob/v1.3-branch/distributions/kfdef/kfctl_openshift_v1.3.0.yaml
```